flutter-studio/src/io/flutter/project/FlutterProjectStep.java

```diff
@@ -94,10 +115,38 @@ public FlutterProjectStep(FlutterProjectModel model, String title, Icon icon, Fl
 
     myValidatorPanel.registerValidator(model.flutterSdk(), FlutterProjectStep::validateFlutterSdk);
     myValidatorPanel.registerValidator(model.projectName(), this::validateFlutterModuleName);
+    myValidatorPanel.registerMessageSource(myDownloadErrorMessage);
 
     Expression<File> locationFile = model.projectLocation().transform(File::new);
     myValidatorPanel.registerValidator(locationFile, PathValidator.createDefault("project location"));
 
+    // Initialization of the SDK install UI was copied from FlutterGeneratorPeer.
+
+    // Hide pending real content.
+    myProgressBar.setVisible(false);
+    myProgressText.setVisible(false);
+    myCancelProgressButton.setVisible(false);
+    myInstallSdkAction = new InstallSdkAction(this);
+
+    myInstallActionLink.setIcon(myInstallSdkAction.getLinkIcon());
+    myInstallActionLink.setDisabledIcon(IconLoader.getDisabledIcon(myInstallSdkAction.getLinkIcon()));
+
+    myInstallActionLink.setText(myInstallSdkAction.getLinkText());
+
+    //noinspection unchecked
+    myInstallActionLink.setListener((label, linkUrl) -> myInstallSdkAction.actionPerformed(null), null);
+
+    myProgressText.setFont(UIUtil.getLabelFont(UIUtil.FontSize.NORMAL).deriveFont(Font.ITALIC));
```

Comment:
I might be mis-seeing but the screenshot makes it look like the progress has a white background...  Can we / should we set it here to be consistent with the wizard background?

Reply:
Perhaps, probably could. But isn't it the same in the IntelliJ wizards?

Reply:
Another thing I wonder about is the visibility of the Install SDK button. I didn't even notice until I was making screenshots that it disappears during the download. That makes more sense in the original layout, I think. But, like progress bar location, it is something we can iterate on.

Reply:
I checked the text box. It has the same appearance in both AS and IJ versions of the wizard. Using the Default theme it has the (questionable) appearance you see in the screenshot. Using the Darkula theme it has a dark charcoal-grey background, as does the dialog box.

Reply:
👍  Thanks for following up!